pgbot/routine.py

```diff
@@ -110,6 +112,102 @@ async def handle_console():
         )
 
 
+async def handle_depression():
+    """
+    Function to handle depression emotion
+    """
+    async with db.DiscordDB("emotions") as db_obj:
+        all_emotions = db_obj.get({})
+
+        try:
+            # Gets the depression emotion dict
+            depression = all_emotions["depression"]
```

Comment:
These 4 lines of try except can be simply replaced with `all_emotions.get("depression", {})`

Reply:
Naisch, fixed it